src/ODB2/odbapi2nc.py.in

```diff
@@ -299,18 +301,18 @@ while row is not None:
     if (refDateTimeString is None):
         refDateTimeString = ioda_conv_util.IntDateTimeToString(row[selectColumns.index(andate_s)], row[selectColumns.index(antime_s)])
     obsDateTimeString = ioda_conv_util.IntDateTimeToString(row[dateIndex], row[timeIndex])
-    #Encode the 8 QC bitfields in the ODB2 file into a single value for IODA
-    #Flip the "active" flags so that zero is always the "good" value for all the flags,
-    #and a qcVal of zero indicates a good datum.
+    # Encode the 8 QC bitfields in the ODB2 file into a single value for IODA
+    # Flip the "active" flags so that zero is always the "good" value for all the flags,
+    # and a qcVal of zero indicates a good datum.
     notReportStatusActive = 0 if row[rsActiveIndex] else 1
     notDatumStatusActive = 0 if row[dsActiveIndex] else 1
     qcVal = (notReportStatusActive * 128 +
-             row[rsPassiveIndex]   *  64 +
-             row[rsRejectedIndex]  *  32 +
-             row[rsBlacklistIndex] *  16 +
-             notDatumStatusActive  *   8 +
-             row[dsPassiveIndex]   *   4 +
-             row[dsRejectedIndex]  *   2 +
+             row[rsPassiveIndex] * 64 +
+             row[rsRejectedIndex] * 32 +
+             row[rsBlacklistIndex] * 16 +
+             notDatumStatusActive * 8 +
+             row[dsPassiveIndex] * 4 +
+             row[dsRejectedIndex] * 2 +
              row[dsBlacklistIndex])
 
```

Comment:
Is there any way to convince the coding norms checker to let us keep the non-standard spacing on lines 308-313? (and the similar lines in odbapi2json). I like the alignment the extra spaces provide as I feel it makes the QC encoding system being implemented a little more clear.

Reply:
We can shut off any error/warning messages we want in the top level .pycodestyle file. The relevant error messages are E221 (multiple spaces before operator) and E222 (multiple spaces after operator).

I'm okay shutting these off since I can't think of what the harm would be for allowing multiple spaces before/after operators. What do others think?

Thanks!

Reply:
@srherbener If special handling of errors is required by a converter, they should add a `.pycodestyle` under their sub-directory.  There is no need to modify the common `.pycodestyle` file at the root of the repository.  What a subset of users like, may not be what others like; hence coding norms.
Simply pass the path to their acceptable `.pycodestyle` in the `CMakeLists.txt` within their sub-directory.  Note however, if the user provided .pycodestyle file is not found, the system one will be used, which may raise more flags.